index.js

```diff
@@ -73,13 +98,13 @@ async function getUrHistories() {
     .slice(0, userNum)
     .reduce((a, [k, v]) => ({ ...a, [k]: v }), {});
 
-  return requiredHis;
+  return [requiredHis, userNum];
 }
 
 // This is testing for exporting
 
 getUrHistories().then((history) =>
-  console.log("here's the finished history", history)
+  console.log(`Here's your top ${history[1]} history command(s)`, history[0])
```

Comment:
We should deconstruct this array. 

const {numberOfCommands, historyOutput} = history
  console.log(`Here's your top ${numberOfCommands} history command(s)`, historyOutput)

Reply:
I deconstructed this array. thanks.